libopenage/util/misc.h

```diff
@@ -66,5 +73,95 @@ struct less {
 	}
 };
 
+
+// Size of uint64_s measured in bytes
+static constexpr size_t uint64_s = 8;
```

Comment:
size of `uint64_t` i assume? And you can do it with `= sizeof(uint64_t)`, as this is the amount of chars, and we use `CHAR_BIT`, so it is consistent and portable.